enhancements/installer/vsphere-ipi-zonal.md

```diff
@@ -252,25 +278,35 @@ type VSpherePlatformNodeNetworking struct {
 	// +optional
 	Internal VSpherePlatformNodeNetworkingSpec `json:"internal"`
 }
-
 // VSpherePlatformSpec holds the desired state of the vSphere infrastructure provider.
 // In the future the cloud provider operator, storage operator and machine operator will
 // use these fields for configuration.
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.apiServerInternalIPs) || has(self.apiServerInternalIPs)",message="apiServerInternalIPs list is required once set"
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.ingressIPs) || has(self.ingressIPs)",message="ingressIPs list is required once set"
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.vcenters) && has(self.vcenters) ? size(self.vcenters) < 2 : true",message="vcenters can have at most 1 item when configured post-install"
 type VSpherePlatformSpec struct {
 	// vcenters holds the connection details for services to communicate with vCenter.
-	// Currently, only a single vCenter is supported.
+	// Currently, only a single vCenter is supported, but in tech preview 3 vCenters are supported.
```

Comment:
I thought one of the epics in 4.18 is to remove techpreview status from multi-vcenter support?

Reply:
I just copied and pasted what the current infra spec is as of 9/9 . afaik it is supposed to be ga'ed in 4.18 but maybe those updates have not been added yet. 

cc: @vr4manta

Reply:
makes sense to merge the logic in an overall doc, but while in flux, I do see the complication of multi vcenter not being here as well.  Currently 4.17 it is feature gated.  Once its all finalized, should be GA.  Goal for GA is 4.18.